extensions/amp-consent/0.1/amp-consent.css

```diff
@@ -85,6 +85,14 @@ amp-consent.amp-hidden {
   animation: 1000ms amp-consent-ui-placeholder-spin linear infinite;
 }
 
+.i-amphtml-consent-alertdialog {
+  overflow: hidden;
+  position: absolute;
+  height: 1px;
+  width: 1px;
+  left: -1;
```

Comment:
nit: -1px. 
And let's make the number bigger in case there's positive padding or margin. 
Besides let's also put `top: auto` based on the doc [here](https://webaim.org/techniques/css/invisiblecontent/#techniques) (omitting `top` may result in the `left` style being ignored. )

Reply:
Good catch